### ISSUE TITLE
Nice document

### ISSUE THREAD

**Issue by susu3621** [AsensingGroup/AsensingDocs#1]
Will there be future updates?

**Comment by luhuadong** [comment]
> Will there be future updates?

Yes, this document should continue to be updated, but unfortunately I have left the company.
